### PR TITLE
Export node as a string

### PR DESCRIPTION
**Pull request by lukicdarkoo**
Another feature to allow users to do reinforcement learning easier. By getting a string of a node users can use it in conjunction with `wb_supervisor_field_import_mf_node_from_string`/`wb_supervisor_field_import_sf_node_from_string` to duplicate environments.

This PR implements:
- `wb_supervisor_node_export_string(node)` which returns content of the node as a string.

**Tasks**
- [x] Add `wb_supervisor_node_export_string` 
- Add support for other languages:
  - [x] C++
  - [x] Python
  - [x] Java
  - [x] Matlab
  - [x] ROS
- [x] Documentation
- [x] Tests
- [x] ROS tests
- [x] Changelog
- [x] Controller.def + WbLanguage.cpp

### CODE REVIEW

File: src/controller/c/supervisor.c [hunk 1]
```diff
@@ -85,6 +85,7 @@ typedef struct WbNodeStructPrivate {
   WbNodeType type;
   char *model_name;
   char *def_name;
+  char *content;
```
Comment (omichel) [inline src/controller/c/supervisor.c:88]: Some housekeeping code is missing for this new field (see my commit).

**Review by omichel — CHANGES_REQUESTED** [review]
The new API function should also be added in `Controller.def` to be properly exported on Windows.

File: src/controller/c/messages.h [hunk 2]
```diff
@@ -92,6 +92,7 @@
 #define C_SUPERVISOR_NODE_ADD_FORCE 70
 #define C_SUPERVISOR_NODE_ADD_FORCE_WITH_OFFSET 71
 #define C_SUPERVISOR_NODE_ADD_TORQUE 72
+#define C_SUPERVISOR_NODE_EXPORT_STRING 75
```
[the remark this answers is not in the document]
Comment (lukicdarkoo) [reply src/controller/c/messages.h:95]: Because in #2740 I use 73 and 74

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**